packages/atomic/src/components/search/facets/atomic-facet/atomic-facet.tsx

```diff
@@ -245,6 +257,13 @@ export class AtomicFacet implements InitializableComponent, BaseFacet<Facet> {
     });
   }
 
+  public componentDidRender() {
```

Comment:
Is it working if you put this emitter in the initialize method? The earlier the better

Reply:
Yup, should work !